x/ccv/provider/client/proposal_handler.go

```diff
@@ -403,13 +331,13 @@ func postConsumerRemovalProposalHandlerFn(clientCtx client.Context) http.Handler
 		)
 
 		msg, err := govtypes.NewMsgSubmitProposal(content, req.Deposit, req.Proposer)
-		if rest.CheckBadRequestError(w, err) {
-			return
-		}
+		//		if rest.CheckBadRequestError(w, err) {
+		//			return
+		//		}
 
-		if rest.CheckBadRequestError(w, msg.ValidateBasic()) {
-			return
-		}
+		//		if rest.CheckBadRequestError(w, msg.ValidateBasic()) {
+		//			return
+		//		}
 
 		tx.WriteGeneratedTxResponse(clientCtx, w, req.BaseReq, msg)
```

Comment:
@sontrinh16 @pysel

hey guys could you please fix up this file?

Reply:
yes i have remove all the rest usage let me clean up the commit and push

Reply:
oh thanks a ton <3